src/libs/pluginprocess/pluginprocess.c

```diff
@@ -6,6 +6,41 @@
  * Executes plugins in a separate process via fork and uses a simple
  * communication protocol based on the dump plugin via named pipes.
  *
+ * The communication protocol works as follows, where Child and Parent stand
+ * for the child and the parent process:
+ * 1)  Two named pipes are created, called commandPipe and resultPipe
+       commandPipe is used for a keyset containing metainformation about
+       the communicationprotocol
+       resultPipe is used for transferring the actual payload that gets passed
+       to a plugin
+ * 2)  The parent constructs a keyset called commandKeySet, containing:
+ *     - /pluginprocess/parent
+ *       a copy of the parent key passed to the plugin interface
+ *     - /pluginprocess/parent/name
+ *       the name of the parent key passed to the plugin interface
+ *     - /pluginprocess/command
+ *       the command that should be executed by the child process
+ *     - /pluginprocess/payload/exists
```

Comment:
What about sending the size instead of a bool? See below.

Reply:
that forces me to convert the number and that is cumbersome in c :)

Reply:
We soon have excellent macros for number conversions. Unfortunately it is not yet merged... #1850